python/tvm/script/ir_builder/relax/ir.py

```diff
@@ -454,6 +454,7 @@ def dtype(value: Union[py_str, DataType]) -> Expr:
     "If",
     "Then",
     "TupleGetItem",
+    "ShapeExpr",
```

Comment:
IIUC the parser supports ShapeExpr in two ways
* the parser is able to parse Python tuples to ShapeExprs, as long as the tuple fields are all PrimExpr with integer dtype.
* the parser has supported `R.shape` https://github.com/tlc-pack/relax/pull/435. I haven’t done a rebase yet to bring this PR to MLC.

Reply:
Let’s go with `R.shape`. Let me know in which case you would need it and if you need a rebase soon.

Reply:
Sure. `R.shape` is a great choice